packages/react-mutation-mapper/src/component/mutationMapper/MutationMapper.tsx

```diff
@@ -444,7 +445,9 @@ export default class MutationMapper<
         return (
             this.store.mutationData.isPending ||
             this.isMutationPlotDataLoading ||
-            this.isMutationTableDataLoading
+            this.isMutationTableDataLoading ||
+            (!!this.store.activeTranscript &&
```

Comment:
i think store should always have activeTranscript, right?

Reply:
because in in `GeneSummary` we use activeTranscript!.result  (https://github.com/cBioPortal/cbioportal-frontend/pull/3211/files#diff-8cec5864c80004992d57fdcaf961294aR365), and activeTrancript is optoinal in MutationMapperStore, so I think we should check activeTrancript here